### ISSUE TITLE
Sample config for COCO with Unet

### ISSUE THREAD

**Issue by sxs4337** [yassouali/pytorch-segmentation#101]
Hi.
Is there a sample config (with the best params), that I can use for UNet training with COCO?

Thanks.

**Comment by zptang1210** [comment]
I tried to write a script for UNet with CityScapes. Looks alright so far. It should be not difficult to adapt it to COCO.

{
    "name": "UNetResnet",
    "n_gpu": 1,
    "use_synch_bn": false,

    "arch": {
        "type": "UNetResnet",
        "args": {
            "backbone": "resnet50",
            "pretrained": true,
            "freeze_bn": false,
            "freeze_backbone": true
        }
    },

    "train_loader": {
        "type": "CityScapes",
        "args":{
            "data_dir": "data/cityscapes",
            "batch_size": 8,
            "split": "train",
            "crop_size": 380,
            "base_size": 400,
            "scale": true,
            "num_workers": 8,
            "mode": "fine",
            "val": false,
            "shuffle": true,
            "augment": true,
            "flip": true,
            "rotate": true,
            "blur": true
        }
    },

    "val_loader": {
        "type": "CityScapes",
        "args":{
            "data_dir": "data/cityscapes",
            "batch_size": 8,
            "crop_size": 480,
            "val": true,
            "split": "val",
            "num_workers": 4
        }
    },

    "optimizer": {
        "type": "SGD",
        "differential_lr": true,
        "args":{
            "lr": 0.01,
            "weight_decay": 1e-4,
            "momentum": 0.9
        }
    },

    "loss": "CrossEntropyLoss2d",
    "ignore_index": 255,
    "lr_scheduler": {
        "type": "Poly",
        "args": {}
    },

    "trainer": {
        "epochs": 80,
        "save_dir": "saved/",
        "save_period": 1,
  
        "monitor": "max Mean_IoU",
        "early_stop": 10,
        
        "tensorboard": false,
        "log_dir": "saved/runs",
        "log_per_iter": 20,

        "val": true,
        "val_per_epochs": 5
    }
}
